assignments/number-functions/code.js

```diff
@@ -0,0 +1,39 @@
+const add =(a,b) =>  {
+  return a+b;
+};
+
+const subtract =(a,b) =>  {
+  return a-b;
+};
+
+const multiply =(a,b) =>  {
+  return a*b;
+}; 
+
+const divide =(a,b) =>  {
+  return a/b;
+};
+
+const mod =(a,b) =>  {
+  return a%b;
+};
+
+const averageOfThree =(a,b,c) =>  {
+  return (a+b+c)/3;
+};
+
+const averageOfTwo =(a,b) =>  {
+  return (a+b)/2;
+};
+
+const distance =(x1,y1) =>  {
+  return Math.abs (x1-y1);
+};
+
+const manhattanDistance =(x1,y1,x2,y2) =>  {
+  return Math.abs (y1-y2) + Math.abs(x1-x2);
```

Comment:
A stylistic note. This is fine but it would arguably be even more clear if you used the `distance` function you just defined. Like:

```suggestion
  return distance(y1, y2) + distance(x1, x2);
```

That makes it super clear to the reader what Manhattan distance is: the sum of the distances on the x- and y-axes.